CONTRIBUTING.md

```diff
@@ -0,0 +1,60 @@
+# Contributing
+
+Thank you for your interest in contributing! Join the [**Plugin Marketplace channel**](https://community.mattermost.com/core/channels/plugins-marketplace) on the Mattermost community server for discussion about this plugin.
+
+## Reporting Issues
+
+If you think you found a bug, [please use the GitHub issue tracker](https://github.com/mattermost/mattermost-marketplace/labels/issues/new) to open an issue.
+
+## Community Plugin
+
+To add your plugin to the Marketplace, please open [an issue using thus template](https://github.com/mattermost/mattermost-marketplace/issues/new?template=add_plugin.md). To update your plugin, please also open an issue [using this template](https://github.com/mattermost/mattermost-marketplace/issues/new?template=update_plugin.md).
+
+### Playbook
+
+This is a playbook used my Mattermost core committers.
+
+#### New plugin
+
+JIRA template:
+  - [PM](https://mattermost.atlassian.net/browse/MM-22224)
+  - [Dev](https://mattermost.atlassian.net/browse/MM-22221)
+  - [Security](https://mattermost.atlassian.net/browse/MM-22225)
+  - [QA](https://mattermost.atlassian.net/browse/MM-22223)
+
+After a new plugin has been submitted, the assignees of the issue posts the following message:
+```
+## Process checklist
+- [ ] Request review and create JIRA tickets and link them here:
+    - PM:
+    - Dev:
+    - Security:
+    - QA:
+- [ ] Fork repo under Mattermost oranisation
+- [ ] Cut plugin release
+- [ ] Add release to Marketplace
+- [ ] Reach out on [Marketing Channel](https://community-release.mattermost.com/private-core/channels/marketing
+) to tweet about the plugin
+- [ ] Work with `@hanna.park` regarding swag
+```
+
+#### Update plugin
+
+JIRA template:
+  - [PM](https://mattermost.atlassian.net/browse/MM-22228)
+  - [Dev](https://mattermost.atlassian.net/browse/MM-22222)
+  - [Security](https://mattermost.atlassian.net/browse/MM-22226)
+  - [QA](https://mattermost.atlassian.net/browse/MM-22227)
+
+After an update for an existing plugin has been submitted, the assignees of the issue posts the following message:
+```
+## Process checklist
+- [ ] Request review and create JIRA tickets and link them here:
+    - PM:
+    - Dev:
+    - Security:
+    - QA:
+- [ ] Update fork
```

Comment:
Should this occur before creating the tickets? Do we create a pull request to review?

Reply:
Because of similar reasons as above I would not update the fork right now. Maybe the contributor even wants to wait with cutting a release him/her self until a reviews are done.

My concern with a pull request is that reviews and conversation might take place in the PR instead of the public GitHub issue.

What is the benefit of a pull request?

Reply:
I think we should streamline the review process for any staff asked to review a process -- having a PR and all the associated tooling makes that easy, otherwise we have to scroll through the diff manually, add inline comments, and mentally keep track of who's approved, etc.

Reply:
Maybe I misunderstand your idea. In my mind  there would be not review process for the fork update. The review of the update should take place in the original GH issues that the community member opened. After every reviewer approved the changes, the fork update can just happen without further reviews. This is made clear in the developers docs: https://github.com/mattermost/mattermost-developer-documentation/pull/524/files#diff-ee526eb501deade82cd69d56e8654e3aR75

Does this makes sense to you?

Reply:
I think it's still a bit unclear to me. As a potential reviewer, here's how I imagined things playing out:
* Community member submits request to update plugin
* Owner updates Mattermost fork with all upstream changes
* Branch created, initially matching last release (i.e. in a matterpoll fork, `mattermost-v1.3.0` is created pointing at the same commit as `mattermost-v.1.2.0`)
* Pull request to merge upstream into `mattermost-v1.3.0` created in fork for internal review
* Any blocking issues result in feedback on original community request / possibly issues on the repository.

I'm inclined to conduct the review in the context of a PR in order to streamline the process for reviewers, provide an internal audit log, discuss security issues, etc. I'm concerned that conducting the review purely in the original issue will require linking out to diffs and ultimately be more challenging to the reviewers time. If changes have to be made, we'll be juggling commit hashes in comments instead of just updating the merge request and re-triggering review.

Reply:
As discussed offline, a pull request is now used to merge the changes to `master`